rules/rules.js

```diff
@@ -87,7 +87,7 @@ const getGroupsForItem = function (ruleConfig) {
  * @returns {Boolean} whether the rule exists
  */
 const ruleExists = function (uid) {
-    return !(ruleRegistry.get(uid) == null);
+    return !(RuleManager.getStatusInfo(uid) == null);
```

Comment:
Out of curiosity, whats the benefit of changing this line ?

Reply:
I think I lost that comment. In UI rules, using the ruleRegistry doesn't work for some reason after the rule is run the second time around. The RuleRegistry gets stale somehow. But the RuleManager works every time.